src/prefect/flows.py

```diff
@@ -42,7 +42,7 @@
 from prefect.futures import PrefectFuture
 from prefect.logging import get_logger
 from prefect.results import ResultSerializer, ResultStorage
-from prefect.server.schemas.core import Flow, FlowRun, raise_on_invalid_name
+from prefect.server.schemas.core import Flow as ORMFlow, FlowRun, raise_on_invalid_name
```

Comment:
To avoid a naming conflict with another `Flow` class below, an alias (`ORMFlow`) is used to import the `Flow` class from `prefect.server.schemas.core`.

Reply:
I'd recommend updating this import to `import prefect.server.schemas as schemas` and then using the module contents with `schemas.core.Flow`. That way you don't need to worry about name collisions and it will also resolve #8688.

Reply:
🕵️‍♂️ Great idea!